doc/man3/OSSL_CMP_CTX_new.pod

```diff
@@ -111,7 +115,7 @@ OSSL_CMP_CTX_set1_senderNonce
  int OSSL_CMP_CTX_set1_referenceValue(OSSL_CMP_CTX *ctx,
                                       const unsigned char *ref, int len);
  int OSSL_CMP_CTX_set1_secretValue(OSSL_CMP_CTX *ctx, const unsigned char *sec,
-                                   const int len);
+                                   int len);
```

Comment:
Please do not change this. Although it might seem stupid, I am afraid this is still an API break.

Reply:
Oops, I had planned but forgot to revert this; done now.